src/deluge/model/global_effectable/global_effectable.cpp

```diff
@@ -183,21 +187,21 @@ bool GlobalEffectable::modEncoderButtonAction(uint8_t whichModEncoder, bool on,
 				currentFilterType =
 				    static_cast<FilterType>((util::to_underlying(currentFilterType) + 1) % kNumFilterTypes);
 
-				char const* displayText;
+				std::string_view displayText;
 				switch (currentFilterType) {
 				case FilterType::LPF:
-					displayText = "LPF";
+					displayText = l10n::getView(STRING_FOR_LPF);
 					break;
 
 				case FilterType::HPF:
-					displayText = "HPF";
+					displayText = l10n::getView(STRING_FOR_HPF);
 					break;
 
 				case FilterType::EQ:
-					displayText = "EQ";
+					displayText = l10n::getView(STRING_FOR_EQ);
 					break;
 				}
-				display->displayPopup(displayText);
+				display->displayPopup(displayText.data());
```

Comment:
maybe out of scope for this change but `Display::displayPopup` should probably take a `string_view` directly.

Reply:
Yeah I was planning on doing that as part of continuing to refactor the displays now that we're post-display unification at some point.